<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcCredit</class>
 <widget class="QWidget" name="CalcCredit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>342</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>342</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>560</width>
    <height>342</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreditCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color:rgb(153, 193, 241);
  border: 1px solid gray;
}</string>
  </property>
  <widget class="QLabel" name="label_answer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="Calculation_option">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
   font: 75 14pt &quot;Ubuntu Mono&quot;;
   background-color:rgb(255, 190, 111);
   border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Ежемесячный платёж</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Переплата по кредиту</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Общая сумма кредита:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Тип платежей:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_percent">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Срок кредита:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_time">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
   background-color:rgb(255, 190, 111);
   border-radius: 10px;
   border: 1px solid gray;
   font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
   font: 75 14pt &quot;Ubuntu Mono&quot;;
   background-color:rgb(255, 190, 111);
   border: 1px solid gray;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Аннуитетные</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Дифференцированные</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Вариант расчёта:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 15pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 14pt &quot;Ubuntu Mono&quot;;
    border: 0px;
}</string>
   </property>
   <property name="text">
    <string>₽, руб.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(143, 240, 164);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 11pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color:rgb(246, 97, 81);
   border-radius: 10px;
   border: 1px solid gray; 
   font: 25 11pt &quot;Umpush&quot;;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	border: 0px
}</string>
   </property>
   <property name="text">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
	border: 0px
}</string>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
